lib/Sema/ConstraintSystem.cpp

```diff
@@ -4403,20 +4405,17 @@ size_t Solution::getTotalMemory() const {
          OpenedPackExpansionTypes.getMemorySize() +
          PackExpansionEnvironments.getMemorySize() +
          size_in_bytes(PackEnvironments) +
+         PackElementGenericEnvironments.size() +
          (DefaultedConstraints.size() * sizeof(void *)) +
          ImplicitCallAsFunctionRoots.getMemorySize() +
-         nodeTypes.getMemorySize() +
-         keyPathComponentTypes.getMemorySize() +
-         size_in_bytes(KeyPaths) +
-         (contextualTypes.size() * sizeof(ASTNode))  +
-         size_in_bytes(targets) +
-         size_in_bytes(caseLabelItems) +
+         nodeTypes.getMemorySize() + keyPathComponentTypes.getMemorySize() +
```

Comment:
I am not sure what happened here but it would be great to remove unrelated formatting changes…

Reply:
Sorry I missed this😅 I'll remove them